src/all.test.js

```diff
@@ -90,6 +90,28 @@ describe('GOV.UK Frontend', () => {
         }, component)
       })
     })
+    it('can be initialised scoped to certain sections of the page', async () => {
+      await page.goto(baseUrl + '/examples/scoped-initialisation', { waitUntil: 'load' })
+
+      // To test that certain parts of the page are scoped we have two similar components
+      // that we can interact with to check if they're interactive.
+
+      // Check that the conditional reveal component has a conditional section that would open if enhanced.
+      await page.waitForSelector('#conditional-not-scoped-1', { visible: false })
```

Comment:
We use a combination of `visible: false`, `visible: true` and `hidden: true` in this test – is there a difference between them, or are they just opposites of each other?

It'd be good to be consistent with either the key (`visible: true`, `visible: false`) or the value (`visible: true`,  `hidden: true`) if so.

Reply:
There's some nuance to it, you can have something in the DOM but not visible, or something not in the DOM and therefore not visible I believe.

I'll try and clarify this.